### DISCUSSION TITLE
ENCODE Motif PWMs

### DISCUSSION THREAD

**Discussion by EraslanBas** [GreenleafLab/ArchR#1928] [Questions / Documentation]
Hi,

I am doing the motif enrichment with the "encode" motif set defined in addMotifAnnotations() function. For human, there are 2,065 TF-motif pairs, some TFs with more than one motif. May I ask how I can get the PWM list of these 2,065 motifs? I checked "http://compbio.mit.edu/encode-motifs/" but it is inaccessible now.

Thank you.

**Comment by rcorces** [comment]
This is the relevant piece of code
https://github.com/GreenleafLab/ArchR/blob/629cf1dedbb378f84389b7618a2a22a5a6ca2214/R/AnnotationPeaks.R#L508-L511

**Reply by EraslanBas** [reply]
Thank you very much Ryan!
